### PR TITLE
Upgrade to Vue 0.12

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request migrate to Vue 0.12.

### CODE REVIEW

File: js/views/post-wizard.vue [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports = {
             }]
          };
     },
-    // paramAttributes: ['orgid'],
+    // props: ['orgid'],
```
Comment (davidbgk) [inline js/views/post-wizard.vue]: Is that really useful to keep commented lines?

File: js/views/topic-wizard.vue [hunk 2]
```diff
@@ -50,7 +50,7 @@ module.exports = {
             }]
          };
     },
-    // paramAttributes: ['orgid'],
+    // props: ['orgid'],
```
Comment (davidbgk) [inline js/views/topic-wizard.vue]: Same here.

**Comment by davidbgk** [comment]
I like the "componentization" path they took. r+wc

File: js/app.vue [hunk 3]
```diff
@@ -243,20 +244,20 @@ module.exports = {
...
-                    Vue.nextTick(cb);
+                    this.$nextTick(cb);
                 }
             } else {
                 require(['./views/' + name + '.vue'], function(options) {
                     self.$options.components[name] = Vue.extend(options);
                     self.view = name;
                     if (callback) {
-                        Vue.nextTick(cb);
+                        self.$nextTick(cb);
```
Comment (davidbgk) [inline js/app.vue]: I wonder if you can `bind(this)` within a `require` to avoid the use of `self`?

File: js/views/post-wizard.vue (hunk 1 above)
Comment (noirbizarre) [inline js/views/post-wizard.vue]: Wasn't meant to. It's search and replace ;)

File: js/views/topic-wizard.vue (hunk 2 above)
Comment (noirbizarre) [inline js/views/topic-wizard.vue]: Wasn't meant to. It's search and replace ;)

**Comment by noirbizarre** [comment]
Yes, this release is more stable, more clear, and more performant !

File: js/app.vue (hunk 3 above)
Comment (noirbizarre) [inline js/app.vue]: Already tried but it's not working right now.

Vue 0.12 introduce Async component, maybe it's the key way to go to.

**Pull request merged**